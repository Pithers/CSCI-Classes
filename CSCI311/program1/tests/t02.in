table table-01 1 Julia
party 1 Lenny 10
end
