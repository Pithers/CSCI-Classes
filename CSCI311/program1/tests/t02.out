Lenny seated at table-01
Lenny finished at table-01
Julia served 1
